x-pack/test/functional/page_objects/lens_page.ts

```diff
@@ -1563,7 +1563,7 @@ export function LensPageProvider({ getService, getPageObjects }: FtrProviderCont
       await retry.waitFor('rendering count to stabilize', async () => {
         const firstCount = await getRenderingCount();
 
-        await common.sleep(1000);
+        await common.sleep(500);
```

Comment:
Could you run flaky test runner to make sure it doesn't cause flakiness?

Reply:
Did a broad flaky test runner with 33x run of most of the Lens suites: https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/7409